apps/media-viewer/src/Mediaviewer.vue

```diff
@@ -14,7 +14,7 @@
         <div class="uk-width-large uk-flex uk-flex-middle uk-flex-center uk-flex-around" style="user-select:none;">
           <oc-icon role="button" class="oc-cursor-pointer" size="medium" @click="prev" name="chevron_left" />
           <!-- @TODO: Bring back working uk-light -->
-          <span class="uk-text-small" style="color:#fff"> {{ activeIndex + 1 }} <span v-translate>of</span> {{ mediaFiles.length }} </span>
+          <span v-if="!folderLoading" class="uk-text-small" style="color:#fff"> {{ activeIndex + 1 }} <span v-translate>of</span> {{ mediaFiles.length }} </span>
```

Comment:
You can add the class `uk-light` which will take care of the text colour.

Reply:
The line above says: 
```html
<!-- @TODO: Bring back working uk-light -->
```
and indeed I don't see light text and it's harder readable than without the manual style. So I'll leave it as it is for now.